### ISSUE TITLE
bing.com: breakage

### ISSUE THREAD

**Issue by TomBarkas** [uBlockOrigin/uAssets#18497]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/124740436/235392297-c4fc6290-73a6-4a70-96a9-3f6a7683f21d.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.bing.com/search?q=how+to+cook+lanb+chops&cvid=fffa7edd3d194149bb68a650c5f81f1c&aqs=edge..69i57j0l8.8401j0j4&FORM=ANAB01&PC=U531`

### Category

breakage

### Description

This happens on a lot of sites with UblockerOrigin extension in Edge (latest version). It happens with no other exensions active. It doesn't happen if I turn off Ublockerorigin extension. This is a test Bing page, but I suspect it happens with all Bing search pages

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>
Copy and pasting the scrnshot seems to give a github address instead of the actuala one. Here it is

https://www.bing.com/search?q=how+to+cook+lanb+chops&cvid=fffa7edd3d194149bb68a650c5f81f1c&aqs=edge..69i57j0l8.8401j0j4&FORM=ANAB01&PC=U531

</details>

### Configuration

<details>

```yaml
uBlock Origin 1.50.0:
 since: 3d.15h.28m
Chromium: 114
filterset (summary):
 network: 95081
 cosmetic: 83978
 scriptlet: 22873
 html: 0
listset (total-discarded, last-updated):
 added:
  fanboy-cookiemonster: 62034-14839, 3d.15h.27m
  ublock-annoyances: 5062-51, 3d.15h.27m
 default:
  user-filters: 0-0, never
  easylist: 67018-49, 3d.15h.27m
  easyprivacy: 30999-56, 3d.15h.27m
  plowe-0: 3733-942, 3d.15h.27m
  ublock-badware: 7329-131, 3d.15h.27m
  ublock-filters: 33495-122, 3d.15h.27m
  ublock-privacy: 397-4, 3d.15h.27m
  ublock-quick-fixes: 201-7, 19h.30m
  ublock-unbreak: 2033-36, 3d.15h.27m
  urlhaus-1: 6023-0, 19h.30m
filterset (user): [empty]
trustedset:
 removed:
  edge-scheme
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 248 ms (selfie)
 maxAssetCacheWait: 125 ms
popupPanel:
 blocked: 25
 network:
  bing.com: 25
 extended:
  ##.ad_sc
  ##.bnp_cookie_banner
  ##.b_ad:style(position: absolute !important; top: -9999px !impor…
  #@#.b_ad
```
</details>

**Comment by okiehsch** [comment]
You manually enabled EasyListCookies/fanboy-cookiemonster and that causes the breakage.
If you enable a cookie list some sites will break, that's why we don't enable them by default.

@ryanbr your filter
`bing.com##.bnp_cookie_banner` makes the site unusable, you can't scroll anymore.
You may have to delete cookies once to be able to reproduce. 
If you still can't reproduce try a EU-IP

**Comment by stephenhawk8054** [comment]
> Copy and pasting the scrnshot seems to give a github address instead of the actuala one

FYI, that's the image link from github now, they change the way of embedding images when uploading a bit than before.

**Comment by stephenhawk8054** [comment]
Actually I can't reproduce even when using EU or non-EU IP or with EasyList Cookies enabled.

**Comment by okiehsch** [comment]
Hm, weird.
I tested a few times and it seems there are two different cookie modals on my end, one looks like this
![image](https://github.com/uBlockOrigin/uAssets/assets/16838044/2deb8be2-e730-4c4d-8c52-1d370cbb33b1)

and if blocked breaks the site.

The otheer modal is
![image](https://github.com/uBlockOrigin/uAssets/assets/16838044/eac86d99-970e-4758-85bc-2fb6fad64ec4)

and you can block that one without any breakage.

Which one shows appears to be random on my end.

**Comment by okiehsch** [comment]
Both are blocked by the same filter, not sure how to handle it.

**Comment by stephenhawk8054** [comment]
Hmmm... I can't see those modals at all on my side.

**Comment by TomBarkas** [comment]
Hello everyone and thanks for rushing to help. I ned to explain I'm completely new to this and this site iand the UBO options are both  quite daunting if you're not a computer expert! Took me a while even to find where to reply.
Okiehsch - where can I find this Fanboy thing? Am I in the right area? ( see screenshot)
I've added bing.com#@#.bnp_cookie_banner to My List and it seems to have sorted it!
Everyone else - is it now sorted or are you testing things>
![G8cRfKNt1P](https://github.com/uBlockOrigin/uAssets/assets/65482253/a4c627b3-69d1-4343-ac49-bb947b00cb0e)

**Comment by okiehsch** [comment]
> Okiehsch - where can I find this Fanboy thing?

What do you mean were can you find it? I can see in your configuration that you have enabled/added it to your filter lists.
To do that you had to manually enable `EasyListCookies` it in your filter lists tab, it's part of that list.

> I'm completely new to this and this site iand the UBO options are both quite daunting if you're not a computer expert! Took me a while even to find where to reply. 

I would suggest to keep the default setup in that case, i.e. don't add any external lists.

**Comment by stephenhawk8054** [comment]
> Even with a EU-IP?

Ah ok, I can see the bottom one now using Germany IP. Weird that last time I used VPN to Switzerland, it did not show up.

**Comment by okiehsch** [comment]
> is it now sorted or are you testing things?

We don't sort it because the issue is caused by an external list, we only fix issues here that are caused by our lists/ the default setup.
If you keep the filter in your "MY List" tab the site should work fine from now on.

**Comment by okiehsch** [comment]
@stephenhawk8054 Switzerland is not part of the [EE](https://en.wikipedia.org/wiki/European_Economic_Area) and they don't have to comply with GDPR.

**Comment by okiehsch** [comment]
@MasterKia no, that does not work.

<details>
<summary>Screenshot(s)</summary>


![image](https://github.com/uBlockOrigin/uAssets/assets/16838044/44e9e940-b201-4fef-88f4-a4baf9e8296d)
</details>

**Comment by stephenhawk8054** [comment]
Ah, so that's why :)

**Comment by krystian3w** [comment]
We can test kiboke team idea:

```adb
bing.com###bnp_cookie_banner[aria-modal="false"]
```

**Comment by ryanbr** [comment]
```
bing.com##.bnp_cookie_banner
bing.com##body:style(overflow: auto !important;)
```
?
